locale/en/blog/release/v10.0.0.md

```diff
@@ -30,18 +30,20 @@ The full set of changes for the Node.js 10.0.0 release are detailed below.
 
 ### Notable Changes
 
-* The N-API native addons API has graduated from experimental status.
```

Comment:
Just wondering about the removal of this one?

Reply:
It is in the list below and I was not sure why it was duplicated here. It is the only entry that is not aligned with the changelog.

Reply:
@BridgeAR I must be missing something.  When I hit "view" and search on graduated I don't see it anywhere after the removal?

Reply:
@BridgeAR thanks, see that is has different wording.